web/src/beta/components/Toggle/index.tsx

```diff
@@ -0,0 +1,57 @@
+import { useCallback } from "react";
+
+import { styled } from "@reearth/services/theme";
+
+export type Props = {
+  checked: boolean;
+  onChange: (checked: boolean) => void;
+  disabled?: boolean;
+};
+
+const Toggle: React.FC<Props> = ({ checked, onChange, disabled = false }) => {
+  const handleClick = useCallback(
+    () => !disabled && onChange(checked),
+    [checked, onChange, disabled],
+  );
+
+  return (
+    <Wrapper>
+      <Switch checked={checked} disabled={disabled} onClick={handleClick}>
+        <TopSlider checked={checked} />
+      </Switch>
+    </Wrapper>
+  );
+};
+
+export default Toggle;
+
+const Wrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`;
```

Comment:
This isn't needed anymore so remove it 🙏🏻

Reply:
Oh yea, sorry :sweat: